### ISSUE TITLE
Event ID 3

### ISSUE THREAD

**Issue by zirdum** [google/cloud-print-connector#429]
I installed this on my 2016 server. Since I installed the service, I'm getting 2 entries a minute for Google Cloud Print Connector in the Application Event Viewer with Event ID 3. Here's what it says:

```
The description for Event ID 3 from source Google Cloud Print Connector cannot be found. Either the component that raises this event is not installed on your local computer or the installation is corrupted. You can install or repair the component on the local computer.

If the event originated on another computer, the display information had to be saved with the event.

The following information was included with the event: 

Printers are already in sync; there are 7
```

It seems to be working fine otherwise. Any ideas what the message is referring to and is there any way to fix this?

Thank you.

**Comment by jay0lee** [comment]
The message from the connector is:

Printers are already in sync; there are 7

this is the regular check to make sure printers and printer settings for
Windows printers are synced with the CloudPrint printers.

This isn't an error.

Jay

**Comment by zirdum** [comment]
Is there a way to make it less verbose? It's filling up the event logs
unnecessarily.

**Comment by eatyourpeas747** [comment]
I agree that the logging needs to be turned down or written to another log file and not the main Application windows log.

**Comment by RixNox** [comment]
Application log got useless as it is only filled up with this message. How can we get rid of this?
